tests/browser-automated-tests-playwright/tests/test_transaction_filter_by_name_and_by_plan_title.js

```diff
@@ -0,0 +1,62 @@
+const playwright = require('playwright');
+const assert = require('assert');
+const PLAYWRIGHT_HOST = process.env.PLAYWRIGHT_HOST;
+const PLAYWRIGHT_HEADLESS = process.env.PLAYWRIGHT_HEADLESS.toLocaleLowerCase() == "true" || false;
+
+/* Test an order can be placed for a plan with subscription & upfront payment */
+async function test_order_plan_with_subscription_and_upfront_charge(browsers, browserContextOptions) {
```

Comment:
- 👌thanks for the comments helps to understand


- Please update the function name 😊